source/Table/types.js

```diff
@@ -25,7 +33,7 @@ export type HeaderRendererParams = {
 
 export type RowRendererParams = {
   className: string,
-  columns: Array,
+  columns: any[],
```

Comment:
hmm I think it just slipped in when I implemented the new types, or perhaps I tried to find an easy to implement more specific type information for this array ;). 
This change on it's own, is more or less a matter of taste any[] and Array should be the same in flowType.